### PR TITLE
added discussionId in ReportDiscussionCommentDetails.js

### LINKED ISSUE #125: add discussionId in the ReportDiscussionCommentDetails Schema
### Current Scenario

The `ReportDiscussionComment` holds and stores only `reason`, `userId` and `discussionCommentId` in its schema.

### Enhancement

Add `discussionId` in the same schema and you can extract it from the path `POST /:discussionId/comment/:commentId` as a route parameter.

### PR DESCRIPTION
**Pull request by ankitvashisht12**
Fixes #125

### CODE REVIEW

File: src/models/ReportDiscussionCommentDetails.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ const mongoose = require('mongoose');
 
 const ReportDiscussionCommentDetailsSchema = new mongoose.Schema(
   {
+    discussionId: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+    },
```
Comment (Dishebh) [inline src/models/ReportDiscussionCommentDetails.js:8]: Add `ref` to `Discussion` model.

**Review by Dishebh — CHANGES_REQUESTED** [review]

File: src/models/ReportDiscussionCommentDetails.js (hunk 1 above)
Comment (ankitvashisht12) [reply src/models/ReportDiscussionCommentDetails.js:8]: sure

File: src/models/ReportDiscussionCommentDetails.js [hunk 2]
```diff
@@ -2,6 +2,11 @@ const mongoose = require('mongoose');
 
 const ReportDiscussionCommentDetailsSchema = new mongoose.Schema(
   {
+    discussionId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Discussion',
+      required: true,
+    },
```
Comment (Dishebh) [inline src/models/ReportDiscussionCommentDetails.js:9]: Please move this block of code below the `userId`

Comment (ankitvashisht12) [reply src/models/ReportDiscussionCommentDetails.js:9]: sure

**Review by Dishebh — APPROVED** [review]

**Pull request merged**